tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -110,13 +110,6 @@ stages:
       parameters:
         xcodeVersion: 14.2
 
```

Comment:
another one to remove:
https://github.com/microsoft/onnxruntime/blob/040bb8c4f8dadea54108f302f18f7da048b866a4/tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml#L397-L403